src/utils/flinkStatementResultColumns.ts

```diff
@@ -0,0 +1,95 @@
+import { SqlV1ResultSchema } from "../clients/flinkSql";
+
+export type ViewMode = "table" | "changelog";
+
+type ColumnDefinition = {
+  index: number;
+  title: () => string;
+  children: (result: Record<string, any>) => any;
+  description: (result: Record<string, any>) => any;
+};
+
+export type ColumnDefinitions = Record<string, ColumnDefinition>;
+
+const OPERATION_TYPES = {
```

Comment:
I agreed with your statement in slack about these not being intuitive on their own. (Insert), (Deletion), etc pattern after the symbol might be helpful?